backend/api/Services/DocumentService.cs

```diff
@@ -74,10 +74,9 @@ public async Task<bool> DeleteActivityDocumentAsync(PimsActivityInstanceDocument
             this.Logger.LogInformation("Deleting PIMS document for single activity");
             this.User.ThrowIfNotAuthorized(Permissions.DocumentDelete);
 
-            IList<PimsActivityInstanceDocument> existingActivityDocuments = documentActivityRespository.GetAllByDocument(activityDocument.DocumentId);
+            IList<PimsActivityInstanceDocument> existingActivityDocuments = documentActivityRespository.GetAllByDocument(activityDocument.DocumentId, true);
```

Comment:
Need to track here, otherwise when we delete we will load each disconnected entity, which will result in the document status code value being loaded multiple times, which will cause an error if the same status code is loaded.